src/main/java/org/ibp/api/ibpworkbench/rest/BreedingView.java

```diff
@@ -117,5 +140,44 @@ public void setBreedingViewService(final BreedingViewService breedingViewService
 		this.breedingViewService = breedingViewService;
 	}
 
+	@ApiOperation(value = "Gets list of Breeding View Licenses")
+	@RequestMapping(value = "/validate-license", method = RequestMethod.POST)
+	@ResponseBody
+	public ResponseEntity<Map<String, Object>> validateBVLicense() {
+		final Map<String, Object> out = new LinkedHashMap<>();
+		HttpStatus isSuccess = HttpStatus.BAD_REQUEST;
+
+		try {
+			final License designLicenseInfo = this.designLicenseService.getLicenseInfo();
```

Comment:
There's already an existing API to get the BV License. Please see BVLicenseResource (GET /breeding-view-licenses). You can use that api to get the license data directly from front-end, and just implement this validation also in the front-end.